### ISSUE TITLE
Does it work with SVG elements?

### ISSUE THREAD

**Issue by ekrausso** [mauricius/vue-draggable-resizable#349]
Hi.. I was wondering if this component could work with SVG elements? Ive been trying with not much luck.. maybe I'm doing something wrong..
 Ive been using it with png images but my images get blurry when too big or small..
thanks!

**Comment by lucianobosco** [comment]
Yes it does. Just be aware that SVG will keep always aspect ratio in case of resize.
